### DISCUSSION TITLE
Problems

### DISCUSSION THREAD

**Discussion by xcentaurix** [OpenCockpit/TMDBCockpit#1] [General]
Please post your problems here...

**Comment by trollysat** [comment]
Hi, nachdem ich über telnet versucht habe zu installieren (mit: wget https://dream-alpha.github.io/TMDBCockpit/tmdbcockpit.sh -O - | /bin/sh) kam folgende Fehlermeldung: 

Connecting to dream-alpha.github.io|2606:50c0:8003::153|:443... connected.
HTTP request sent, awaiting response... 404 Not Found                     
2024-01-23 16:03:37 ERROR 404: Not Found.     

Wie kann ich das plugin installieren? DM 920 mit OE2.5 instable

**Comment by xcentaurix** [comment]
hi trollysat, 
probiere bitte nochmal... habe das problem gefixt.
dream-alpha

**Reply by trollysat** [reply]
Danke für die schnelle Antwort, es funzt!

**Comment by trollysat** [comment]
Über das Menü kann man ja TMDB auf die gelbe Taste bei EPGs legen lassen. Leider funktioniert das nicht beim Merlin EPG Center, kann man das fixen?

**Comment by xcentaurix** [comment]
ich leider nicht. das muesste jemand machen, der sich mit dem code des merlin epg centers auskennt.
